app/src/components/Input/index.tsx

```diff
@@ -0,0 +1,23 @@
+import React, { useState } from 'react';
+import { StyledInput } from './style';
+
+interface Props {
+  type: string;
+  placeholder: string;
+  value: string;
+}
+
+const Input = ({ type = 'text', placeholder = '', value = '' }: Props) => {
+  const [Value, setValue] = useState(value);
+
+  return (
+    <StyledInput
+      type={type}
+      placeholder={placeholder}
+      value={Value}
+      onChange={(e) => setValue(e.target.value)}
+    />
```

Comment:
```suggestion
    <StyledInput
      type={type}
      placeholder={placeholder}
      value={value}
      onChange={onChange}
    />
```

- input 컴포넌트의 onChange와 state는 부모컴포넌트에서 관리해야 합니다.  props로 value와 onChange를 받을 수 있게끔 수정 부탁드립니다.
- 또한, useState의 Value 네이밍을 value로 수정해주세요.